frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -265,6 +267,13 @@ export default function CreateClusterPage() {
                 control.available = curatorTemplates.map((template) => template.metadata.name)
                 setAvailableTemplates(control, curatorTemplates)
                 break
+            case 'singleNodeFeatureFlag':
+                if (
+                    featureGateCache.find((fg) => fg.metadata.name === 'open-cluster-management-single-node-openshift')
```

Comment:
I approved too early lol, can we import the enum from `FeatureGates.ts` and reference here instead of hard coding the string

Reply:
Sure.  Change pushed.